docs/getting_started/rules.md

```diff
@@ -209,20 +214,20 @@ The following snippet is a simple example on checking specific property values:
 def check_value(self, value, path):
     """Check SecurityGroup descriptions"""
     matches = []
-    full_path = ('/'.join(str(x) for x in path))
 
     # Check max length
     if len(value) > 255:
         message = 'GroupDescription length ({0}) exceeds the limit (255) at {1}'
+      full_path = '/'.join(str(x) for x in path)
```

Comment:
Looks good in general -- can you just correct this indentation issue and I can merge.

Reply:
Fixed!